valhalla/baldr/pathlocation.h

```diff
@@ -104,6 +87,12 @@ struct PathLocation : public Location {
     else if (pl.stoptype_ == Location::StopType::BREAK_THROUGH)
       l->set_type(odin::Location::kBreakThrough);
 
+    l->set_preferred_side(odin::Location::either);
+    if (pl.preferred_side_ == Location::PreferredSide::SAME)
+      l->set_preferred_side(odin::Location::same);
+    else if (pl.preferred_side_ == Location::PreferredSide::OPPOSITE)
+      l->set_preferred_side(odin::Location::opposite);
```

Comment:
bug, the tests didnt test this so it was skipped. now they do, and they failed when it wasnt here.